spec/cob_index/macros/custom_spec.rb

```diff
@@ -1401,6 +1401,47 @@
         expect { subject.map_record(records[14]) }.not_to raise_error
       end
     end
+
+    context "when all items are inactive" do
+      record_text = <<-EOT
+<record>
```

Comment:
@dkinzer I haven't seen us use this before: record_text = <<-EOT.  What is the benefit of that?  Also, the indentation seems off for the record text fields in all of the new examples.

Reply:
@sensei100 it's basically like "  but for situations where you want to use some copy-pasta text (like for a fixture) that can may have a combo of single and double quotes but you don't want to bother figuring out how/where to do the quote escapes.

Reply:
Oh, the indentation, ... when you use this quote (I think it's called heredoc?) It's an indicator that the text that follows will  use original format (including spacing).

So another use case is for creating templates.  But basically anything between the heredoc entry and exit codes does not need to be formatted in a specific way.  The linter will know this and not throw an error.